examples/HelloApp/app/src/main/java/com/github/kright/helloapp/MainActivity.java

```diff
@@ -1,9 +1,9 @@
 package com.github.kright.helloapp;
 
 import android.os.Bundle;
-import android.support.v7.app.AppCompatActivity;
+import android.app.Activity;
```

Comment:
use AppCompatActivity; u need to install Android Support Library to compile it though